es61x/src/main/scala/tech/beshu/ror/es/ssl/SSLNetty4InternodeServerTransport.java

```diff
@@ -33,34 +32,33 @@
 import org.elasticsearch.indices.breaker.CircuitBreakerService;
 import org.elasticsearch.threadpool.ThreadPool;
 import org.elasticsearch.transport.netty4.Netty4Transport;
-import scala.collection.JavaConverters$;
-import tech.beshu.ror.configuration.SslConfiguration;
 import tech.beshu.ror.configuration.SslConfiguration.InternodeSslConfiguration;
-import tech.beshu.ror.utils.SSLCertParser;
+import tech.beshu.ror.utils.SSLCertHelper;
 
 import javax.net.ssl.SSLEngine;
 import javax.net.ssl.TrustManagerFactory;
-import java.io.InputStream;
 import java.net.SocketAddress;
 import java.security.AccessController;
 import java.security.PrivilegedAction;
 import java.util.Optional;
-import java.util.stream.Collectors;
 
 public class SSLNetty4InternodeServerTransport extends Netty4Transport {
 
   private final Logger logger = LogManager.getLogger(this.getClass());
   private final InternodeSslConfiguration ssl;
+  private final Boolean fipsCompliant;
```

Comment:
I wonder if we shouldn't add the flag to InternodeSslConfiguration? (the same applies to SslConfiguration). WDYT?

Reply:
I wanted to make `FipsCompliant` setting part of `SslConfiguration`. However there are plans to make ROR fully compatible with FIPS. Fully compliant FIPS would require SSL FIPS + some additional things. In that case I thought it would be probably better to have general configuration parameter for FIPS mode.